.github/template/prepare-checks/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: prepare-checks
+description: Prepares everything for static code check execution
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Cache Go dependencies
+      uses: actions/setup-go@v4
+      with:
+        go-version-file: './go.mod'
+        check-latest: true
+        cache-dependency-path: './go.sum'
+
+    # uses an older go version by default, so configure go with newest version
+    - name: setup golang
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.21'
```

Comment:
Let's unify names and descriptions in workflows and steps in all GH config files. I have no preferences, let's just make it consistent. For example, start from upper-case, the rest of the sentence is in lower-case:
```
- name: Setup go
- description: Deploy operator in release mode
etc...
```

Reply:
done